keras_cv/models/segmentation/fcn.py

```diff
@@ -0,0 +1,355 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+from keras_cv.models import VGG16
+from keras_cv.models import VGG19
+from keras_cv.layers.preprocessing import Resizing
+
+BACKBONE_CONFIG = {
```

Comment:
Use layer names instead of magic numbers

Reply:
Makes sense. I was actually trying to slice and get all the layers from the beginning upto that specific layer. Will do that, then